--- a/dev/null	2023-07-25 07:15:04.422447910 +0800
+++ b/mw2fcitx/utils/manual_fix.json	2022-05-03 21:54:07.938199718 +0800
@@ -0,0 +1,4 @@
+{
+  "朝之琉璃": "zhao'zhi'liu'li",
+  "朝之瑠璃": "zhao'zhi'liu'li"
+}

--- a/dev/null	2023-07-25 07:15:04.422447910 +0800
+++ b/mw2fcitx/utils/manual_fix.py	2022-05-03 21:54:07.938199718 +0800
@@ -0,0 +1,19 @@
+import json
+from os import path
+
+from .logger import console
+
+MANUAL_FIX_TABLE = {}
+JSON_PATH = path.join(path.dirname(path.realpath(__file__)), "manual_fix.json")
+
+try:
+    MANUAL_FIX_TABLE = json.loads(open(JSON_PATH).read())
+    console.debug("Manual fix table loaded.")
+except Exception as e:
+    console.debug(f"Error loading manual fix table: {str(e)}")
+
+
+def manual_fix(text):
+    if text in MANUAL_FIX_TABLE:
+        return MANUAL_FIX_TABLE[text]
+    return None

